sdv/metadata/multi_table.py

```diff
@@ -545,7 +564,26 @@ def visualize(self, show_table_details=True, show_relationship_labels=True,
                     'primary_key': f'Primary key: {table_meta.primary_key}'
                 }
 
-        else:
+        elif show_table_details == 'summarized':
+            default_sdtypes = ['id', 'numerical', 'categorical', 'datetime', 'boolean']
+            for table_name, table_meta in self.tables.items():
+                count_dict = defaultdict(int)
+                for column_name, meta in table_meta.columns.items():
+                    sdtype = 'other' if meta['sdtype'] not in default_sdtypes else meta['sdtype']
+                    count_dict[sdtype] += 1
+
+                count_dict = dict(sorted(count_dict.items()))
+                columns = ['Columns']
+                columns.extend([
+                    fr'&nbsp; &nbsp; • {sdtype} : {count}'
+                    for sdtype, count in count_dict.items()
+                ])
+                nodes[table_name] = {
+                    'columns': r'\l'.join(columns),
+                    'primary_key': f'Primary key: {table_meta.primary_key}'
+                }
+
+        elif show_table_details is None:
```

Comment:
Need an error if `show_table_details` is something else, otherwise it would raise a weird error. Could maybe validate in on top of the file.